docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -804,6 +804,46 @@ recommended to use one of the other modes and rely on
 to inject your custom configuration.
 </Admonition>
 
+## `podMonitor`
+
+`podMonitor` controls [the PodMonitor CR (from monitoring.coreos.com/v1)
+](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.PodMonitor)
+This custom resource is used to configure Prometheus to monitor workload running
```

Comment:
I think we can improve this help text - at no point does it really say that it'll enable monitoring of the workloads deployed by this chart 😅